<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admin-PC" ignored="0" name="Verify Authentication process of a finsys user having role of ITM" tests="2" failures="2" timestamp="24 Jul 2018 17:06:45 GMT" time="1.169" errors="0">
  <testcase name="@AfterTest ExecuteCloseApplication" time="0.0" classname="ej.finsys.modules.CloseApplication">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ej.finsys.modules.CloseApplication.ExecuteCloseApplication(CloseApplication.java:14)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest ExecuteCloseApplication -->
  <testcase name="ExecuteLaunchApplication" time="0.019" classname="ej.finsys.modules.LaunchApplication">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at ej.finsys.modules.LaunchApplication.ExecuteLaunchApplication(LaunchApplication.java:117)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ExecuteLaunchApplication -->
  <testcase name="ExecuteLoginToApplication" time="1.021" classname="ej.finsys.modules.LoginToApplication">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at ej.finsys.modules.LoginToApplication.ExecuteLoginToApplication(LoginToApplication.java:60)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ExecuteLoginToApplication -->
</testsuite> <!-- Verify Authentication process of a finsys user having role of ITM -->
